### DISCUSSION TITLE
mortal的文档还会继续更新吗？

### DISCUSSION THREAD

**Discussion by hrbattery** [Equim-chan/Mortal#35] [Q&A] [answered]
我留意到mortal的文档[https://mortal.ekyu.moe](https://mortal.ekyu.moe)还有许多的章节没有更新，其中包括如何训练、mjai、libriichi的文档等等。我正在分析本地自行训练模型并用于批量分析自己的牌谱的可行性。我阅读过先前关于麻将ai顾虑的issue（[https://gist.github.com/Equim-chan/cf3f01735d5d98f1e7be02e94b288c56](https://gist.github.com/Equim-chan/cf3f01735d5d98f1e7be02e94b288c56)），能理解对于公开训练模型方法方面的顾虑，但请问其他章节的文档是否仍有更新的计划？
谢谢。

**Comment by Equim-chan** [comment, marked as answer]
目前没有继续更新文档的计划。

**Reply by hrbattery** [reply]
会接受文档更新相关的pr吗？

**Reply by Equim-chan** [reply]
其实不太想更新文档的一个原因就是我对文档的要求比较高，但是我现在拿不出什么精力和动力去做这个事情。
